### PR TITLE
AAP-19350

### PR DESCRIPTION
**Pull request by g-murray**
AAP-19350  correcting incorrect steps in the install guide. I also reformatted steps to highlight mandatory steps from optional.

https://issues.redhat.com/browse/AAP-19350

### CODE REVIEW

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 1]
```diff
@@ -143,37 +137,26 @@ ldap_extra_settings:
...
-Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
-+
 [NOTE]
 ====
 If it is not practical to re-run `setup.sh` or if debug logging is enabled for a short time, you can add a line containing `GALAXY_LDAP_LOGGING: True` manually to the `/etc/pulp/settings.py` file on {PrivateHubName}. Restart both `pulpcore-api.service` and `nginx.service` for the changes to take effect. To avoid failures due to human error, use this method only when necessary.
 ====
-+
-. Use this example to configure LDAP caching by setting the variable `AUTH_LDAP_CACHE_TIMEOUT`.
-+
+* Use this example to configure LDAP caching by setting the variable `AUTH_LDAP_CACHE_TIMEOUT`.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:152]:
```suggestion
* Use this example to configure LDAP caching by setting the variable `AUTH_LDAP_CACHE_TIMEOUT`.
+
```

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 2]
```diff
@@ -45,38 +45,39 @@ automationhub_ldap_group_search_base_dn = "ou=people,dc=ansible,dc=com"
...
-. Create a yaml file that contains `ldap_extra_settings`. 
+. Create a YAML file that contains `ldap_extra_settings`. 
 * Example:
 +
 ----
 #ldapextras.yml
 ---
 ldap_extra_settings:
-  AUTH_LDAP_USER_ATTR_MAP: '{"first_name": "givenName", "last_name": "sn", "email": "mail"}'
+ <LADP_parameter>: <Values>
```
Comment (sayjadha) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:73]: Typo, suggest `LADP parameter` -> `LDAP parameter`

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 3]
```diff
@@ -45,38 +45,39 @@ automationhub_ldap_group_search_base_dn = "ou=people,dc=ansible,dc=com"
...
+
 ...
 ----
 +
-Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+. Add any parameters that you require for your set up. The following examples  describe the LDAP parameters that you can set in `ldap_extra_settings`:
```
Comment (sayjadha) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:78]: As this is a noun (setup) and not a verb (set up) here, suggest **set up** -> **setup**. References of usage are available in the Supplementary style guide here: https://redhat-documentation.github.io/supplementary-style-guide/.

**Review by sayjadha — COMMENTED** [review]
Minor edits here, rest LGTM. Good catch with the 'superuser' usage; it took me a minute to find the usage so I learnt something new today :-)

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 3 above)
Comment (g-murray) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:78]: Good spot :)

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 2 above)
Comment (g-murray) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:73]: Thank you!

**Review by sayjadha — COMMENTED** [review]
LGTM.

**Pull request merged**